templates/password-change.tpl

```diff
@@ -0,0 +1,28 @@
+<div id="edit_form">
+<form name="mailbox" method="post">
+<table>
+   <tr>
```

Comment:
This file should use tabs instead of spaces for indentations to be consistent with other templates.